cves/kernel/CVE-2020-14386.yml

```diff
@@ -155,10 +173,13 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    A developer from Palo Alto Networks discovered and reported the
+    vulnerability. The developer, Or Cohen, was looking into the packet
+    sockets in the Linux kernel source code when he found the vulnerability.
+  automated: false
+  contest: false
+  developer: false
```

Comment:
This should probably be true since a developer found the vulnerability.

Reply:
the prompt says "...then determine if the vulnerability was found by a Google employee"  so I assumed it meant a Google or Linux employee, but I'll switch to true